test/UnitTest/Microsoft.AspNet.OData.Test/Microsoft.AspNet.OData.Test.csproj

```diff
@@ -10,8 +10,8 @@
     <RootNamespace>Microsoft.AspNet.OData.Test</RootNamespace>
     <AssemblyName>Microsoft.AspNet.OData.Test</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <OutputPath>..\..\..\bin\$(Configuration)\UnitTest\</OutputPath>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\</SolutionDir>
+    <OutputPath>$(WebStackRootPath)bin\$(Configuration)\UnitTest\AspNet\</OutputPath>
```

Comment:
This causes the VSTS build to no longer find the test binary. I modified the VSTS task to work for the old and new path by adding \**\.

Reply:
These tests are running in VSTS.